packages/web-pkg/src/composables/actions/spaces/useSpaceActionsRestore.ts

```diff
@@ -28,15 +28,11 @@ export const useSpaceActionsRestore = ({ store }: { store?: Store<any> } = {}) =
     const client = clientService.graphAuthenticated
     const promises = spaces.map((space) =>
       client.drives
-        .updateDrive(
-          space.id.toString(),
-          { name: space.name },
-          {
-            headers: {
-              Restore: true
-            }
+        .updateDrive(space.id.toString(), {} as any, {
```

Comment:
as any is not optimal but sending an empty object is not valid in the api spec, but the request goes trough